packages/server/package.json

```diff
@@ -22,11 +22,12 @@
     "build:clean": "rimraf ./dist && yarn build",
     "dev:clean": "yarn build:clean && yarn dev",
     "dev:server:test": "cross-env DISABLE_NOTIFICATIONS_CONSUMPTION=true NODE_ENV=test LOG_LEVEL=silent LOG_PRETTY=true node ./bin/ts-www",
-    "test": "cross-env NODE_ENV=test LOG_LEVEL=silent LOG_PRETTY=true mocha",
+    "ts-mocha": "node --require ts-node/register ./bin/mocha",
```

Comment:
moving the `ts-node/register` require to happen as early as possible, not while mocha is already running, seems to fix the issue